lab-andrew/json/reminder.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You should delete this file.  PRs should not have anything unnecessary, comments, files, folders, etc.